src/java/fr/paris/lutece/portal/service/workflow/IWorkflowProvider.java

```diff
@@ -278,4 +281,19 @@ List<Integer> getAuthorizedResourceList( String strResourceType, int nIdWorkflow
      */
     String doValidateTasksForm( int nIdResource, String strResourceType, int nIdAction, HttpServletRequest request, Locale locale, User user );
 
+    /**
+     * Get the HTML confirmation of action processing 
+     * 
+     * @param nIdAction
+     *            the action id
+     * @param locale
+     *            the locale
+     * @param actionHistoryResourceIdList
+     *            the ids of the history of the action performed on a resource
+     * @return the confirmation
+     */
+    default String getDisplayProcessActionConfirmation( int nIdAction, Locale locale, List<Integer> actionHistoryResourceIdList )
```

Comment:
thanks for the correction,
would it be possible to take this method out of the CORE, to manage the message display mode only in forms?

Reply:
as it has been requested, this method is placed in the core so that it can be used in other plugins such as appointement and not only in forms

Reply:
This new feature allows to make available to the different plugins the return messages of the execution of the tasks, but it is then up to each plugin to exploit these messages according to their environment.
The doProcessAction method only returns one message for example.

Reply:
The requested change has been implemented